### PR TITLE
VPS-131/adding self to contributors readme (John)

### PR DESCRIPTION
**Pull request by codecreator127**
## Describe the issue

John's profile missing from ReadMe contributor's list

## Describe the solution

Added John to the ReadMe contributor's list by copying existing table data and editing the profile image and names. Put next to Tech Lead Woo-Jin's profile.

## Risk

Potential merging conflicts

## Definition of Done

- [ ] Code peer-reviewed
- [x] Acceptance criteria met

## Reviewed By

Who reviewed your PR - for commit history once merged

### CODE REVIEW

File: yarn.lock
Comment (JordanBlenn) [inline yarn.lock:1]: The `package.json`, `package-lock.json`, and `yarn.lock` should be regenerated every time you run `yarn run init` (I believe), so you shouldn't need to commit them.

File: README.md
Comment (JordanBlenn) [inline README.md:1]: Looks good :)

**Review by JordanBlenn — CHANGES_REQUESTED** [review]
Good - I would prefer you remove the lock files from the PR just in terms of simplicity, but it shouldn't really change the functionality of the code as they get regenerated anyways (see other comments).

File: yarn.lock
Comment (codecreator127) [reply yarn.lock:1]: will remove them. Going to add those to the gitignore to avoid future such issues

**Review by JordanBlenn — APPROVED** [review]
All good

**Pull request closed without merging**